lessons/Shiny/lesson.md

```diff
@@ -0,0 +1,320 @@
+#R Shiny Tutorial
+
+ - **Authors**: Lanna Jin
+ - **Research field**: Ecology & Evolutionary Biology
+ - **Lesson Topic**: Interactive Visualizations with the Shiny library in R
```

Comment:
It's technically called a 'package' not a 'library', which is something else. Shiny is a 'package' while a 'library' is where all the packages are stored. Packages are like books, which are found in the library. Same concept.

It's a fairly common misconception, so no worries.